etherman/feijoa_event_sequence_blobs.go

```diff
@@ -151,12 +149,25 @@ func parseBlobCallDataTypeParams(data []byte) (*BlobCommonParams, []byte, error)
 	//		zkGasLimit           uint64
 	//		l1InfoLeafIndex      uint32
 	//	    transactions         []byte
-	buf := bytes.NewBuffer(data)
-	result := &BlobCommonParams{}
-	err := binary.Read(buf, binary.LittleEndian, result)
+
+	// Prepare blob params using ABI encoder
+	uint64Ty, _ := abi.NewType("uint64", "", nil)
+	uint32Ty, _ := abi.NewType("uint32", "", nil)
+	bytesTy, _ := abi.NewType("bytes", "", nil)
+	arguments := abi.Arguments{
+		{Type: uint64Ty},
+		{Type: uint64Ty},
+		{Type: uint32Ty},
+		{Type: bytesTy},
+	}
+	unpacked, err := arguments.Unpack(data)
```

Comment:
I think this could be done using anonymous structs and using marshall/unmarshall
https://www.willem.dev/articles/anonymous-structs/

Reply:
What do you mean? instead of using **abi.Arguments**?